bionic/persistence.py

```diff
@@ -765,6 +769,10 @@ def search(self, url_prefix):
             for sub_path in path_prefix.glob("**/*")
         ]
 
+    def delete(self, url):
+        path = path_from_url(url)
+        path.unlink()
```

Comment:
This throws an error if the path doesn't exist. It would be nice if this is more robust (or maybe the caller `delete_entry`) and either catches that exception and warns or passes `missing_ok=True` so it doesn't throw an exception.